angular/demo/daisyui/src/app/samples/toast/toast.component.ts

```diff
@@ -0,0 +1,43 @@
+import {BaseWidgetDirective, type ToastWidget, type ToastProps, auBooleanAttribute, callWidgetFactory, createToast} from '@agnos-ui/angular-headless';
+import {ChangeDetectionStrategy, Component, EventEmitter, Input, Output, inject} from '@angular/core';
+import closeIcon from '@agnos-ui/common/samples/common/close_icon.svg';
+import {DomSanitizer} from '@angular/platform-browser';
+
+@Component({
+	selector: 'app-toast',
+	template: `
+		@if (!state().hidden) {
+			<div class="alert {{ state().className }} flex">
+				<ng-content />
+				@if (state().dismissible) {
+					<button
+						class="btn btn-sm btn-circle btn-ghost"
+						(click)="api.close()"
+						[attr.aria-label]="state().ariaCloseButtonLabel"
+						[innerHTML]="closeIcon"
+					></button>
+				}
+			</div>
+		}
+	`,
+	standalone: true,
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ToastComponent extends BaseWidgetDirective<ToastWidget> {
+	private readonly sanitizer = inject(DomSanitizer);
+	readonly closeIcon = this.sanitizer.bypassSecurityTrustHtml(closeIcon);
+	@Input({transform: auBooleanAttribute})
+	visible: ToastProps['visible'];
+	@Input({transform: auBooleanAttribute})
+	dismissible: ToastProps['dismissible'];
+	@Input() ariaCloseButtonLabel: ToastProps['ariaCloseButtonLabel'];
+	@Input() className: ToastProps['className'];
+	@Output() visibleChange = new EventEmitter<boolean>();
+	@Output() hidden = new EventEmitter<void>();
+	@Output() shown = new EventEmitter<void>();
+	readonly _widget = callWidgetFactory({
```

Comment:
very minor, but I would put some empty lines between closeIcon and the inputs / outputs, and between the inputs / outputs and the `readonly _widget` line. looks slightly better in the demo site.